db/migrate/20221104161158_create_foods.rb

```diff
@@ -0,0 +1,14 @@
+class CreateFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name
+      t.string :description
+      t.string :image
+      t.string :price
+      t.string :category
+      t.string :rating
+      t.string :order_id
+      t.timestamps
```

Comment:
Kindly add the user_id references here and the index because the admin will create the food as well. for that we need to associate the food model to the user model